rnd/autogpt_server/autogpt_server/schema.prisma

```diff
@@ -0,0 +1,67 @@
+datasource db {
+  provider = "sqlite"
+  url      = "file:./database.db"
+}
+
+generator client {
+  provider  = "prisma-client-py"
+  interface = "asyncio"
+}
+
+// This model describe a single node in the Agent Graph/Flow (Multi Agent System).
+model AgentNode {
+  id String @id
```

Comment:
is id a uuid or incremental id? imo uuid better overall because it prevents some heavy assumptions people make about incremental ids